### ISSUE TITLE
Add an credential definition based on any existing/published schema on the same ledger

### ISSUE THREAD

**Issue by cvarjao** [bcgov/traction#657]

**Comment by loneil** [comment]
So this would be adding a cred def without having to make yourself (the tenant) a **copy** of a schema and then make the cred def based on that schema?

**Comment by loneil** [comment]
Ok, we could probably pretty easily have a separate "create cred def" that takes a schema rather than the only method in the UI now which is to create a def from one of the schemas in your list of strored schemas.
Though I think the UX around creating and managing your cred defs is pretty confusing right now so would wrap this up in whatever re-think of that is needed,

**Comment by esune** [comment]
Yes, the UX should be reviewed and updated. I agree with you that cred def is a "tricky" concept.
For now, we could just input the schema id (as on the ledger) by hand, if it is not picked from the list of ids stored in the wallet.

**Comment by hiteshgh** [comment]
This feature is already implemented , according @swcurran
